workflow-templates/knative-go-build.yaml

```diff
@@ -52,3 +52,6 @@ jobs:
 
       - name: Build
         run: go build -v ./...
+
+      - name: Build Test
+        run: for d in $(go list -tags e2e -f '{{if .TestGoFiles}}{{.Dir}}{{end}}' ./... ); do (echo $d; cd $d && go test -tags e2e -c ); done
```

Comment:
We have a bunch more tags (e.g. `performance`, `postupgrade`, `preupgrade`) I think there's a script in our wad of bash to extract and pass them all.

Reply:
except the one we use for `hack/tools.go` IIRC